python/sglang/bench_one_batch.py

```diff
@@ -509,6 +510,8 @@ def latency_test(
     bench_args,
     tp_rank,
 ):
+    initialize_moe_config(server_args)
```

Comment:
This is a hack; I opened this PR hoping that someone knows a better way to initialize the MOE config?